src/framework/piral-core/src/utils/foreign.test.ts

```diff
@@ -0,0 +1,43 @@
+import { createElement, createRef } from 'react';
+import { convertComponent, renderInDom } from './foreign';
+import { ForeignComponent } from '../types';
+import { DefaultLoadingIndicator } from '../components/DefaultLoader';
+
+describe('Util Foreign.', () => {
+  it('Convert component function throws error due to missing converter function.', () => {
+    try {
+      expect(convertComponent(null, createElement('div'))).toThrow(
+        'No converter for component of type "div" registered.',
+      );
+    } catch {
+      expect(true).toEqual(true);
+    }
+  });
+  it('Convert component function returns converted component.', () => {
```

Comment:
Would appreciate a blank line between the tests - just to be able to see immediately what's going on.

Thanks!

Reply:
Done. Thanks for hint.